src/presentation/http/router/userList.ts

```diff
@@ -0,0 +1,58 @@
+import type { FastifyPluginCallback } from 'fastify';
+import type UserListService from '@domain/service/userList.js';
+
+/**
+ * Interface for the userList router
+ */
+interface UserListRouterOptions {
+    /**
+     * User list service instance
+     */
+    userListService: UserListService,
+
+}
```

Comment:
bad code style

Reply:
how can I fix it? can you show the pattern of code style? I should remove comments or reduce the number of spaces in the lines?

Reply:
we use 2 space indentation. Install Editor Config plugin to IDE to respect the .editorconfig file